src/types/guilds/membership_screening.ts

```diff
@@ -0,0 +1,12 @@
+import { SnakeCaseProps } from "../util.ts";
+
+export interface MembershipScreening {
+  /** When the fields were last updated */
+  version: string;
+  /** The steps in the screening form */
+  formFields: MembershipScreeningField[];
+  /** The server description shown in the screening form */
+  description: string | null;
+}
+
```

Comment:
Missing "doc resource" link here

Reply:
Does not exist as for now. They removed the documentation. But it still exists so should we keep the type?

Reply:
If it is not documented, it would be better to just remove it entirely and we can implement it later with changes from Discord.